Examples/ArcGISToolkitExamples/Misc/ARStatusTableViewController.swift

```diff
@@ -0,0 +1,146 @@
+            return "Limited - Initializing"
+        default:
+            return ""
+        }
+    }
+}
+
+/// A view controller for display AR-related status information.
+class ARStatusTableViewController: UITableViewController {
+    
+    /// The `ARKit` camera tracking state.
+    public var trackingState: ARCamera.TrackingState = .notAvailable {
+        didSet {
+            DispatchQueue.main.async{ [weak self] in
+                self?.tableView.reloadData()
+            }
+        }
+    }
+    
+    /// The calculated frame rate of the `SceneView` and `ARKit` display.
+    public var frameRate: Int = 0 {
+        didSet {
+            DispatchQueue.main.async{ [weak self] in
+                self?.tableView.reloadData()
+            }
+        }
+    }
+    
+    /// The current error message.
+    public var errorMessage: String = "None" {
+        didSet {
+            DispatchQueue.main.async{ [weak self] in
+                self?.tableView.reloadData()
+            }
+        }
+    }
+
+    /// The label for the currently selected scene.
+    public var currentScene: String = "None" {
+        didSet {
+            DispatchQueue.main.async{ [weak self] in
+                self?.tableView.reloadData()
+            }
+        }
+    }
+    
+    /// The translation factor applied to the current scene.
+    public var translationFactor: Double = 1.0 {
+        didSet {
+            DispatchQueue.main.async{ [weak self] in
+                self?.tableView.reloadData()
+            }
+        }
+    }
+    
+    /// The labels for each status item.
+    private let cellLabels = ["Tracking State", "Frame Rate", "Error", "Scene", "Translation Factor"]
+    
+    /// The height of our rows.
+    private let rowHeight: CGFloat = 24.0
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Set the rowHeight of our table view.
+        tableView.rowHeight = rowHeight
+        
+        // Add a blur effect behind the table view.
+        tableView.backgroundView = UIVisualEffectView(effect: UIBlurEffect(style: .regular))
+    }
+    
+    /// Calculates and returns the height of the table view based on the row height and number of rows.
+    ///
+    /// - Returns: The calculated height of the table view.
+    public func height() -> CGFloat {
+        return CGFloat(cellLabels.count) * rowHeight
+    }
+    
+    // MARK: - Table view data source
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return (section == 0) ? cellLabels.count : 0
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        // Don't reuse cells, as our table is essentially static.
+        let statusCell = UITableViewCell(style: .value1, reuseIdentifier: nil)
```

Comment:
Creating a new cell each time circumvents the cell queueing mechanism built into `UITableView`. And since the table view is being reloaded each time a property changes, that means that five old cells are discarded and five new cells are created rather than the same five cells being reused each time. I would suggest either creating a xib for the cell or a `UITableViewCell` subclass and registering that with a reuse identifier with the table view in `viewDidLoad()`. Then, dequeue that cell here using `dequeueReusableCell(withIdentifier:for:)`.

Side Note: You could register `UITableViewCell` with a reuse identifier, but since the cell style is passed as part of the initializer and it can't be changed, that would always get you the basic style. So either a xib or a subclass (requesting the desired style when calling `super`) is needed to get a non-basic cell.